<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="reCheckBox">
       <property name="text">
        <string>RE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ereCheckBox">
       <property name="text">
        <string>ERE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="o2CheckBox">
       <property name="text">
        <string>O2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="minTempCheckBox">
       <property name="text">
        <string>Min Temp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="maxTempCheckBox">
       <property name="text">
        <string>Max Temp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="minRhCheckBox">
       <property name="text">
        <string>Min RH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="maxRhCheckBox">
       <property name="text">
        <string>Max RH</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="removeAllPushButton">
       <property name="text">
        <string>&amp;Remove All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plotAllPushButton">
       <property name="text">
        <string>&amp;Plot All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetScalePushButton">
       <property name="text">
        <string>Reset &amp;Scale</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
